README.md

```diff
@@ -114,6 +114,7 @@
 - [MrStanDu33](https://github.com/MrStanDu33/MrStanDu33)
 - [Jayraj Roshan](https://github.com/jayrajroshan/jayrajroshan)
 - [amxchang](https://github.com/amxchang/amxchang)
+- [ridermansb](https://github.com/Ridermansb)
```

Comment:
Change it from `https://github.com/Ridermansb` to `https://github.com/Ridermansb/Ridermansb`

Reply:
Fixed